ports/frozen-bubble/gameinfo.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<gameList>
+  <game>
+    <path>./frozen-bubble.sh</path>
```

Comment:
.sh Name needs to be Uppercase. This is the Name that is shown when no Metadata is used.